models/User.js

```diff
@@ -20,6 +21,10 @@ const userSchema = mongoose.Schema({
 userSchema.pre('save', function (done) {
     var user = this;
 
+    if (!validator.isEmail(user.email)) {
```

Comment:
Use Mongoose Validation for a cleaner code
[https://github.com/leepowellcouk/mongoose-validator](https://github.com/leepowellcouk/mongoose-validator)